data_api/api/models.py

```diff
@@ -216,7 +216,7 @@ def fetch_objects(cls, org):
         objs = cls.sync_temba_objects(org, ls)
         if not ls:
             ls = LastSaved.create_for_org_and_collection(org, cls)
-        ls.last_saved = datetime.now(tz=pytz.timezone(org.timezone))
+        ls.last_saved = datetime.utcnow()
```

Comment:
mongo seems to drop the timezone information anyways so this is a bit more explicit